Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,153 @@
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imeating",
+                    category=['you'],prompt="I'm Eating/Drinking",pool=True,unlocked=True,
+                    mas_getEV('monika_imeating').shown_count))
+if seen_event(monika_imeating) = False:
```

Comment:
Should be more like:
```renpy
init 5 python:
    addEvent(Event(persistent.event_database,eventlabel="monika_imeating",category=['you'],prompt="I'm Eating/Drinking",pool=True,unlocked=True)

label monika_imeating:
    if mas_getEV('monika_imeating').shown_count == 0:
        #first time dialogue here

    #Now, instead of falling through to the other label, just move everything from there into this label.